go.mod

```diff
@@ -209,6 +209,7 @@ require sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6 // indirect
 
 require (
 	github.com/Sirupsen/logrus v0.0.0-00010101000000-000000000000 // indirect
+	github.com/agext/levenshtein v1.2.3 // indirect
```

Comment:
👀  what is this?

Reply:
Indirect dependency coming from -> `github.com/moby/buildkit/frontend/dockerfile/instructions`
It's used to suggest misspells on commands, e.g. `FRON -> FROM`